src/utils/hooks/usePermissions/usePermissions.ts

```diff
@@ -19,57 +14,43 @@ export type UsePermissions = () => {
 
 const usePermissions: UsePermissions = () => {
   const { t } = useKubevirtTranslation();
-  const cdiInstalled = useFlag(FLAG_KUBEVIRT_CDI);
+  const [activeNamespace] = useActiveNamespace();
```

Comment:
`activeNamespace` can be `ALL_NAMESPACES_SESSION_KEY` in case of `all-namespaces`. In that case, what should we do?